rfc6962-bis.xml

```diff
@@ -399,7 +399,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
         </section>
         <section title="redactedSubjectAltName Certificate Extension" anchor="redacted_san_extension">
           <t>
-            The redactedSubjectAltName extension is a non-critical extension (OID 1.3.101.77) that is identical in structure to the subjectAltName extension, except that dNSName identities MAY contain redacted labels (see <xref target="redacting_labels"/>).
+            The redactedSubjectAltName extension is a non-critical extension (OID 1.3.101.77) that is identical in structure to the subjectAltName extension, except that dNSName identities (and only dNSName identities) MAY contain redacted labels (see <xref target="redacting_labels"/>).
```

Comment:
Instead of adding "(and only dNSName identities)" here, I suggest adding the following paragraph at the end of this section:

"Only DNS-IDs can be redacted using this mechanism. However, CAs can use Name Constraints (section 4.3) to allow DNS domain name labels in other types of subjectAltName entry to not appear in logs."

Conveniently, section 4.3 follows immediately afterwards.